README.md

```diff
@@ -16,6 +16,15 @@ gradient-free optimizers, and constrained optimization.
 * OpenBLAS or Intel MKL or LAPACK (see Armadillo site for details)
 
 
+### Installation
```

Comment:
It seems to me like we can leave this as part of the `Requirements` section, so maybe this line isn't needed?

Reply:
Is CMake strictly required though?  If I'm not mistaken, ensmallen can be used without installation.

A user could just simply copy the files from `ensmallen/include` to wherever they like, or even just use the `ensmallen/include` directory directly during compilation.  Example:
```
  g++ prog.cpp -o prog -O2 -I ~/git/ensmallen/include -larmadillo
```

Reply:
Oh, huh, good point.  It is required to build the tests, but you're right, they could do exactly what you specified.  I retract my comment then. :)

Reply:
ha :) does this imply that the american date format (MM/DD/YY) bugs you as well?